src/generate.ml

```diff
@@ -230,22 +250,23 @@ let types is_ec2 shapes =
               (fun_
                  "xml"
                  (app1
-                    "Util.option_all"
+                    "Aws.Util.option_all"
                     (app2
                        "List.map"
                        (ident (shp ^ ".parse"))
-                       (app2 "Xml.members" (str item_name) (ident "xml"))))))
+                       (app2 "Aws.Xml.members" (str item_name) (ident "xml"))))))
       | Shape.Enum _opts ->
           Syntax.(
             let_
               "parse"
               (fun_
                  "xml"
                  (app2
-                    "Util.option_bind"
-                    (app1 "String.parse" (ident "xml"))
-                    (fun_ "s" (app2 "Util.list_find" (ident "str_to_t") (ident "s"))))))
+                    "Aws.Util.option_bind"
+                    (fun_ "s" (app2 "Aws.Util.list_find" (ident "str_to_t") (ident "s")))
+                    (app1 "String.parse" (ident "xml")))))
```

Comment:
The order of arguments for `bind` need to be swapped around here `(app1 "String.parse" (ident "xml"))` should come first.